.gitignore

```diff
@@ -14,6 +14,10 @@ UserInterfaceState.xcuserstate
 $RECYCLE.BIN/
 ionic.config.json
 .npmrc
+mock/local-journal.json
+mock/local-journal-non-test-activities.json
+src/assets/mock/local-journal.json
```

Comment:
Can you please add the `src/assets/mock` folder all together as it's basically copied from the `mock` folder